2021-05-09 01:27:39,876 INFO    MainThread:25307 [wandb_setup.py:_flush():69] setting env: {}
2021-05-09 01:27:39,877 INFO    MainThread:25307 [wandb_init.py:_log_setup():336] Logging user logs to /home/trenton-windows/repos/spicy-dann/wandb/offline-run-20210509_012739-3o5hkej8/logs/debug.log
2021-05-09 01:27:39,877 INFO    MainThread:25307 [wandb_init.py:_log_setup():337] Logging internal logs to /home/trenton-windows/repos/spicy-dann/wandb/offline-run-20210509_012739-3o5hkej8/logs/debug-internal.log
2021-05-09 01:27:39,877 INFO    MainThread:25307 [wandb_init.py:init():369] calling init triggers
2021-05-09 01:27:39,877 INFO    MainThread:25307 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2021-05-09 01:27:39,877 INFO    MainThread:25307 [wandb_init.py:init():418] starting backend
2021-05-09 01:27:39,877 INFO    MainThread:25307 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-05-09 01:27:39,889 INFO    MainThread:25307 [backend.py:ensure_launched():132] starting backend process...
2021-05-09 01:27:39,895 INFO    MainThread:25307 [backend.py:ensure_launched():137] started backend process with pid: 25334
2021-05-09 01:27:39,896 INFO    MainThread:25307 [wandb_init.py:init():423] backend started and connected
2021-05-09 01:27:39,899 INFO    MainThread:25307 [wandb_init.py:init():465] updated telemetry
2021-05-09 01:27:39,902 INFO    MainThread:25307 [wandb_init.py:init():522] starting run threads in backend
2021-05-09 01:27:43,240 INFO    MainThread:25307 [wandb_run.py:_console_start():1565] atexit reg
2021-05-09 01:27:43,241 INFO    MainThread:25307 [wandb_run.py:_redirect():1412] redirect: SettingsConsole.REDIRECT
2021-05-09 01:27:43,241 INFO    MainThread:25307 [wandb_run.py:_redirect():1417] Redirecting console.
2021-05-09 01:27:43,243 INFO    MainThread:25307 [wandb_run.py:_redirect():1479] Redirects installed.
2021-05-09 01:27:43,243 INFO    MainThread:25307 [wandb_init.py:init():546] run started, returning control to user process
2021-05-09 01:27:43,245 INFO    MainThread:25307 [wandb_run.py:_config_callback():787] config_cb None None {'dataset': 'camelyon17', 'data_root': './WILDS', 'model_name': 'resnet18', 'batch_size': 3, 'n_epochs': 1, 'get_train_metrics': True, 'save_every': 5, 'run_name': 'dev-small', 'max_val_batches': 2}
2021-05-09 01:31:13,048 INFO    MainThread:25307 [wandb_run.py:_atexit_cleanup():1535] got exitcode: 1
2021-05-09 01:31:13,051 INFO    MainThread:25307 [wandb_run.py:_restore():1507] restore
2021-05-09 01:31:15,079 INFO    MainThread:25307 [wandb_run.py:_wait_for_finish():1657] got exit ret: 
2021-05-09 01:31:15,184 INFO    MainThread:25307 [wandb_run.py:_wait_for_finish():1657] got exit ret: done: true
exit_result {
}

